lib/optimize/SplitChunksPlugin.js

```diff
@@ -174,7 +189,9 @@ module.exports = class SplitChunksPlugin {
 								name: option.name,
```

Comment:
for backward compat you need to take care of the `(typeof option === "function")` case too and add something like:

``` js
if (result.chunks) result.chunksFilter = SplitChunksPlugin.normalizeChunksFilter(result.chunks);
```